repository_service_tuf/cli/admin/metadata.py

```diff
@@ -0,0 +1,532 @@
+# SPDX-FileCopyrightText: 2022-2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+import json
+from datetime import datetime, timedelta
+from typing import Any, List
+
+import click
+from rich import box, markdown, prompt, table
+from securesystemslib.exceptions import StorageError  # type: ignore
+from tuf.api.metadata import Metadata, Root
+from tuf.api.serialization import DeserializationError
+
+from repository_service_tuf.cli import console
+from repository_service_tuf.cli.admin import admin
+from repository_service_tuf.cli.admin.ceremony import _load_key, _save_payload
```

Comment:
We could do a structure change here:
- move `_load_key` function to `repository_service_tuf.helpers.tuf` as public `load_key`. Other CLI features could use it (`ceremony`, `metadata update` and [`key public-info`](https://github.com/repository-service-tuf/repository-service-tuf-cli/issues/290)) 
- move `_save_payload` function to `repository_service_tuf.helpers.tuf` as public `save_payload`. Other CLI features could use it

Reply:
About the first bullet, I implemented it in the below PR. You can use it from there.
- https://github.com/repository-service-tuf/repository-service-tuf-cli/pull/293